k8s/create-kind.sh

```diff
@@ -2,16 +2,18 @@
 
 echo "Provisioning Kind"
 kind create cluster --name m3cluster --config kind-cluster.yaml
+export KUBECONFIG="$(kind get kubeconfig-path --name="m3cluster")"
 echo "installing dashboard"
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
 echo "creating service account"
 kubectl create serviceaccount dashboard -n default
 kubectl create clusterrolebinding dashboard-admin -n default --clusterrole=cluster-admin --serviceaccount=default:dashboard
-# pre-load MinIO, postgres, etcd, etcd-operator to speed up setup
+# pre-load MinIO, postgres, etcd, etcd-operator, Prometheus to speed up setup
 docker pull minio/minio:RELEASE.2019-12-17T23-16-33Z
 docker pull postgres:12
 docker pull quay.io/coreos/etcd-operator:v0.9.4
 docker pull quay.io/coreos/etcd:v3.4.0
+docker pull quay.io/prometheus/prometheus:v2.14.0
```

Comment:
you should also load the prometheus image into kind with `kind load docker-image quay.io/prometheus/prometheus:v2.14.0`

Reply:
done